pom.xml

```diff
@@ -305,6 +305,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-invoker-plugin</artifactId>
```

Comment:
@jrdalpra I believe, we're using four spaces for indentation, in the entire document. didn't you see it?